internal/oci/annotations.go

```diff
@@ -11,6 +11,7 @@ const (
 
 	StateDriverPath   = "remoteproc.resolved-path"
 	StateFirmwarePath = "remoteproc.firmware"
+	StateStoredFWName = "remoteproc.stored-firmware"
```

Comment:
Please aim to keep things consistent. It helps save precious brain tokens otherwise spent figuring out why the puzzle pieces don't fit on occasion:

- `StateFirmwarePath`, but then `StateStoredFWName` - we should refer to "firmware" consistently
- We used to have `StateXxxYyy` that mapped to `remoteproc.xxx-yyy`, this is not the case anymore